src/sentry/api/endpoints/organization_integration_serverless_functions.py

```diff
@@ -40,7 +40,7 @@ def get(self, request: Request, organization, integration_id) -> Response:
         return self.respond(serverless_functions)
 
     def post(self, request: Request, organization, integration_id) -> Response:
-        integration = self.get_integration(organization, integration_id)
+        integration = self.get_integration(organization.id, integration_id)
```

Comment:
integration is now an ORM, organization is an Rpc, but we still need to change using the `id` because OrganizationIntegrationl.organization_id is no longer a relationship.